server/fakedb/Event.csv

```diff
@@ -0,0 +1,12 @@
+﻿createdAt,updatedAt,hostId,title,description,price,maxParticipants
```

Comment:
Nice! Shouldn't there also be the location of the event?

Reply:
We thought there could be multiple places for one event. So, we created table for storing events, which is `Schedule` db.